src/kafka/clients.clj

```diff
@@ -0,0 +1,94 @@
+(ns kafka.clients
+  (:gen-class)
+  (:import
+    (java.time Duration)
+    (java.net InetAddress)
+    (org.apache.kafka.common.serialization StringSerializer StringDeserializer)
+    (org.apache.kafka.clients.producer KafkaProducer ProducerRecord)
+    (org.apache.kafka.clients.consumer KafkaConsumer)))
+
+
+(def properties
+  (atom {:producer-config {"client.id",         (.getHostName (InetAddress/getLocalHost))
+                           "group.id",          "group1"
+                           "bootstrap.servers", "localhost:9092"
+                           "key.serializer"     StringSerializer
+                           "value.serializer"   StringSerializer}
+         :consumer-config {"client.id",         (.getHostName (InetAddress/getLocalHost))
+                           "group.id",          "group1"
+                           "bootstrap.servers", "localhost:9092"
+                           "key.deserializer"   StringDeserializer
+                           "value.deserializer" StringDeserializer}}))
+
+(def clients (atom {:producer [] :consumer []}))
```

Comment:
Why set initially to vectors?

Reply:
Because we might build multiple clients right? so we can directly populate the new producer/consumer into the right category

Reply:
And you would store these as an ordered list?

Reply:
The names furthermore imply singularity...